src/lib/util.ml

```diff
@@ -613,3 +613,8 @@ let close_output_with_check (o, temp_file_name, opt_dir, file_name) =
   let do_replace = !always_replace_files || not (same_content_files temp_file_name file_name) in
   let _ = if not do_replace then Sys.remove temp_file_name else move_file temp_file_name file_name in
   ()
+
+let to_upper_camel_case (s : string) =
+  let ss = List.concat (List.map (String.split_on_char '-') (String.split_on_char '_' s)) in
```

Comment:
`String.split_on_char '_' s |> List.map (String.split_on_char '-') |> List.concat` would be the way to write this with the OCaml `|>` combinator.

Usually we don't put type annotations on arguments in `.ml` files if there's a signature in the `.mli`.